### ISSUE TITLE
Store path in URL parameters

### ISSUE THREAD

**Issue by alexsb** [cnobre/cnobre.github.io#5]
This will allow you to save and share the project.

**Comment by cnobre** [comment]
you say "store path in url parameters", do you mean the path to the dataset?

**Comment by alexsb** [comment]
No,  the actual path. See e.g. https://www.google.com/maps/dir/John+Harvard's,+Cambridge,+MA/33+Oxford+St,+Cambridge,+MA+02138,+USA/@42.3756806,-71.1200018,17z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x89e37742ceb8a909:0x7666471e31ddf163!2m2!1d-71.119381!2d42.372568!1m5!1m1!1s0x89e377411efe059d:0x9adc69aff51b57fd!2m2!1d-71.1173546!2d42.3787956!3e2

**Comment by cnobre** [comment]
oh the user defined path! got it
